src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -2,9 +2,12 @@ package racingcar.domain
 
 class Cars(carNames: List<String>) : List<Car> by (carNames.map { name -> Car(name) }) {
 
-    fun winners(attemptCount: Int): List<Car> {
-        val maxPosition = maxByOrNull { car -> car.currentPosition(attemptCount) }
-            ?.currentPosition(attemptCount)
-        return filter { car -> car.currentPosition(attemptCount) == maxPosition }
+    fun winners(): List<Car> {
+        val maxPosition = maxOfOrNull { it.currentPosition() }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val maxPosition = maxOf { it.currentPosition() }
```